GuiFunctions/Databases/DownloadUniProtDatabaseFunctions.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using static UsefulProteomicsDatabases.ProteinDbRetriever;
+
+namespace GuiFunctions.Databases
+{
+    /// <summary>
+    /// We place functions from the GUI in the class to enable testing.
+    /// </summary>
+    
+    public class DownloadUniProtDatabaseFunctions
+    {
+
+        public static string GetUniProtHtmlQueryString(string proteomeID, bool reviewed, bool isoforms, bool xmlFormat, bool compressed)
+        {
```

Comment:
This could/should be refactored with a string builder, but it's not pressing.

Reply:
Done

Reply:
Sent a PM. I think we can do this method without nested conditionals.